tests/src/test/scala/DogsSuite.scala

```diff
@@ -8,33 +8,33 @@ import catalysts.Platform
 
 import cats._
 
-import org.scalatest.{FunSuite, PropSpec, Matchers, BeforeAndAfterAll}
+import org.scalatest.{FunSuite, Matchers, BeforeAndAfterAll}
 import org.scalatest.prop.{Configuration, GeneratorDrivenPropertyChecks}
 import org.typelevel.discipline.scalatest.Discipline
 
-import org.scalacheck.{Arbitrary, Gen}
-
-import scala.util.{Failure, Success, Try}
-
 trait TestSettings extends Configuration with Matchers {
 
   lazy val checkConfiguration: PropertyCheckConfiguration =
     if (Platform.isJvm)
-      if(scala.sys.env.get("TRAVIS").isDefined)
-        PropertyCheckConfiguration(minSuccessful = 3, maxDiscardedFactor = 500F, minSize = 0, sizeRange = 50, workers = 4)
+      if(scala.sys.env.get("SCOVERAGEON").isDefined)
+        PropertyCheckConfiguration(minSuccessful = 1, maxDiscardedFactor = 10F, minSize = 0, sizeRange = 3, workers = 1)
```

Comment:
I'm not sure - It will mean the code will not be hammered by as many random inputs, that's for sure. But scoverage is just about "did this statement get called once".